docs/content/installation/installation-with-manifests.md

```diff
@@ -90,7 +90,7 @@ If you would like to use the TCP and UDP load balancing features of the Ingress
     $ kubectl apply -f common/crds/k8s.nginx.org_globalconfigurations.yaml
     ```
 
-> **Feature Status**: The Policy resources are available as a preview feature[^1]: We might introduce some backward-incompatible changes to the resource definition. The feature is disabled by default.
+> **Feature Status**: The OIDC Policy is available as a preview feature[^1]: We might introduce some backward-incompatible changes to the resource definition. The feature is disabled by default.
```

Comment:
We should remove the preview wording here - OIDC is no longer preview with this change. We can change the wording to the be the same as the policy resource docs above, or remove it altogether. @brianehlert thoughts?

Reply:
It technically has not been a preview feature for a long time, just hidden behind a preview flag.
I am fine with removing the language.
Especially with the deprecation behavior.

Reply:
Done, thanks!